### ISSUE TITLE
Multiple Workspace on Self-Host Openblocks

### ISSUE THREAD

**Issue by nkristianto** [openblocks-dev/openblocks#215]
Hi,
First of all, thank you for the great tool.
Want to know if there is a way to have multiple workspaces on self-host openblocks?

Thanks

**Comment by lmx1117** [comment]
![image](https://user-images.githubusercontent.com/116335720/215236890-69cb50e9-5194-41e9-885a-7c4e9355f354.png)
Sure, you can create a new workspace here

**Comment by neon-balcony** [comment]
Hey @nkristianto, thanks for the post. Could you please describe your scenario? FYI, one of our planned features in the future would be to have multiple workspaces that serves a common "organization", so that you can manage members under the "organization" and then workspace is only for developers creating apps.

**Comment by nkristianto** [comment]
hi, @neon-balcony thanks for the response. The use cases are something like this:
say the company has Finance, Tech, and HR. Is it possible to organize the tools under the different workspaces for every division?

**Comment by neon-balcony** [comment]
@nkristianto I think for now you can move members from each division into different user groups, and then assign app permissions to each specific user group. As a result end users will only see their authorized apps.

**Comment by nkristianto** [comment]
@neon-balcony for the paid version, do you guys already have the pricing model?
is there a feature for SSO, LDAP, and audit log on paid version?

**Comment by neon-balcony** [comment]
@nkristianto Yes we support enterprise SSO login like CAS/OAuth 2.0(OIDC)/LDAP for our enterprise edition, if you're interested to try out the paid version, please find this guy on [Discord](https://discord.com/channels/1040505649798332437/1040509691433857036), he is our sales manager 😄 
<img width="455" alt="image" src="https://user-images.githubusercontent.com/118174126/216314461-29dbb8ad-8bfd-45ab-91cb-90fd061942e5.png">
